src/solidity-frontend/solidity_convert.cpp

```diff
@@ -4020,6 +4036,39 @@ bool solidity_convertert::get_expr(
       callee_expr_json.contains("nodeType") &&
       callee_expr_json["nodeType"] == "MemberAccess")
     {
+      const auto &base = callee_expr_json["expression"];
+
+      if (base.contains("referencedDeclaration"))
+      {
+        int base_decl_id = base["referencedDeclaration"];
+        const auto &decl = find_decl_ref(src_ast_json, base_decl_id);
+
+        if (
+          decl["nodeType"] == "ContractDefinition" &&
+          decl["contractKind"] == "library")
+        {
+          std::string lib_name = decl["name"];
+          std::string func_name = callee_expr_json["memberName"];
+
+          const auto &func_def = get_func_decl_ref(lib_name, func_name);
+          if (func_def.is_null())
+          {
+            log_error(
+              "Cannot locate function '{}' in library '{}'",
+              func_name,
+              lib_name);
+            return true;
+          }
+
+          side_effect_expr_function_callt call;
+          if (get_non_library_function_call(func_def, expr, call))
```

Comment:
What if there are arguments in the function call?

Reply:
Oh i see you are using get_non_library_function_call. so there should be no issues then